### PR TITLE
OSDOCS-6006-2: various nits MicroShift 4-13

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-6006

Link to docs preview:
http://file.rdu.redhat.com/shdiaz/OSDOCS-6006-2/microshift_configuring/microshift-cluster-access-kubeconfig.html#remote-access-con_microshift-kubeconfig
http://file.rdu.redhat.com/shdiaz/OSDOCS-6006-2/microshift_configuring/microshift-cluster-access-kubeconfig.html#accessing-microshift-cluster-remotely_microshift-kubeconfig
http://file.rdu.redhat.com/shdiaz/OSDOCS-6006-2/welcome/index.html

QE review:
N/A internal docs copy edits only

### CODE REVIEW

File: welcome/index.adoc [hunk 1]
```diff
@@ -57,7 +57,7 @@ Next, view the xref:../microshift_release_notes/microshift-4-13-release-notes.ad
 
 * For information about Red Hat Device Edge, read the link:https://access.redhat.com/documentation/en-us/red_hat_device_edge/4/html/overview/device-edge-overview[Red Hat Device Edge overview].
 * For information about Red Hat Enterprise Linux for Edge, read link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images/index[RHEL for Edge documentation].
-* For documentation that is not specific to {product-title}, read link:https://docs.openshift.com/container-platform/latest/welcome/index.html[{product-title} documentation].
+* For container platform documentation that is not specific to {product-title}, read link:https://docs.openshift.com/container-platform/latest/welcome/index.html[OpenShift Container Platform documentation].
```
Comment (GroceryBoyJr) [inline welcome/index.adoc:60]: @ShaunaDiaz I'm confused about why the {product-title} varible would not have worked for the link text?

**Review by GroceryBoyJr — COMMENTED** [review]
@ShaunaDiaz only one tiny nit.

File: welcome/index.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply welcome/index.adoc:60]: @GroceryBoyJr I couldn't see an obvious reason, so I perhaps it is a distro difference?

File: modules/microshift-accessing-cluster-remotely.adoc [hunk 2]
```diff
@@ -48,9 +48,7 @@ Use the following procedure to access the {product-title} cluster from a remote
 $ chmod go-r ~/.kube/config
 ----
 
-.Verify {product-title} is running
-
-. As `user@workstation`, verify that {product-title} is running by running the following command:
+. As `user@workstation`, verify that {product-title} is running by entering the following command:
```
Comment (michaelryanpeter) [inline modules/microshift-accessing-cluster-remotely.adoc:51]: Nit. Typically we format this as a verification step according to the Modular Documentation guide. Take a look at the [procedure template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/main/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc) for an example. This is not a blocker.

```suggestion
.Verification
* As `user@workstation`, verify that {product-title} is running by entering the following command:
```

File: welcome/index.adoc (hunk 1 above)
Comment (michaelryanpeter) [inline welcome/index.adoc:60]: Nit: Typically, when we have to link across branches/distros, we use a link to the Customer Portal over docs.openshift.com. That said, that rule is not documented and is probably worth bringing up with the team to clarify after GA. Not a blocker.

**Review by michaelryanpeter — COMMENTED** [review]
I left two small nits that can be investigated after GA.

File: modules/microshift-accessing-cluster-remotely.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply modules/microshift-accessing-cluster-remotely.adoc:51]: Correct, I had it this way before but it looked weird with a #1 step procedure and and #1 step verification. Can we have a one-step verification without the step number?

File: welcome/index.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply welcome/index.adoc:60]: Oh, right. That's just template text I didn't dig into.

File: modules/microshift-accessing-cluster-remotely.adoc (hunk 2 above)
Comment (michaelryanpeter) [reply modules/microshift-accessing-cluster-remotely.adoc:51]: We can use an astericks for single-step procedures.

**Pull request merged**